shared/src/ledger/native_vp/ethereum_bridge/bridge_pool_vp.rs

```diff
@@ -82,6 +84,51 @@ where
         }
     }
 
+    /// Check that the correct amount of wrapper erc20s were
+    /// sent from the correct account into escrow
+    fn check_werc20s_escrowed(
```

Comment:
Such a nit, but `werc` looks funny to me. Can we just call the function `check_erc20s_escrowed`?